geometry_msgs/Vector3 pos
geometry_msgs/Vector3 vel
geometry_msgs/Vector3 acc
